API.md

```diff
@@ -126,6 +126,27 @@ Display a Media on the left / right part of a Paragraph
 />
 ```
 
+<h2 name="box">Box</h2>
+
+Display a generic rectangle shape with customisable width, height, color and borderRadius
+
+#### Props available
+- `height: Number | String`: the height of the component (default: `40`)
+- `width: Number | String`: the width of the component (default: `50`)
+- `borderRadius: Number`: the border radius of the component (default: `0`)
```

Comment:
What do you think of only using `radius` instead of `borderRadius` ? It's a smaller prop name, and in my thoughts, it's self descriptive ?